### PR TITLE
Console-Iot - Update create project component

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Refactoring
### Description
- Include Changes of PF update in endpoint configuration steps

### CODE REVIEW

File: console/console-init/ui/src/modules/project/components/EndpointConfiguraion/EndpointConfiguraion.tsx [hunk 1]
```diff
@@ -146,74 +128,64 @@ const EndpointConfiguration: React.FunctionComponent<IEndpointConfigurationProps
...
+          {getProtocolOptions().map(protocol => (
+            <div key={`key-protocol-${protocol.key}`}>
+              <br />
+              <Checkbox
+                label={protocol.label}
+                isChecked={isProtocolPresent(protocol.value)}
+                onChange={onProtocolChange}
+                value={protocol.value}
+                aria-label={`Protocol checkbox to select ${protocol.value}`}
+                id={`checkbox-${protocol.key}`}
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/modules/project/components/EndpointConfiguraion/EndpointConfiguraion.tsx:148]: Stick to the convention for naming IDs

File: console/console-init/ui/src/modules/project/components/EndpointConfiguraion/EndpointConfiguraion.tsx [hunk 2]
```diff
@@ -146,74 +128,64 @@ const EndpointConfiguration: React.FunctionComponent<IEndpointConfigurationProps
...
-              onChange={onRouteChange}
-            />
-          </FormGroup>
-        </Form>
-      </GridItem>
-    </Grid>
+      <FormGroup fieldId="form-group-create-routes" label="Create Routes">
+        <br />
+        <SwitchWithToggle
+          id="switch-configure-route-btn"
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/modules/project/components/EndpointConfiguraion/EndpointConfiguraion.tsx:182]: Add component name as prefix.

File: console/console-init/ui/src/modules/project/components/EndpointConfiguraion/EndpointConfiguraion.tsx [hunk 3]
```diff
@@ -146,74 +128,64 @@ const EndpointConfiguration: React.FunctionComponent<IEndpointConfigurationProps
...
+          {getCertificateOptions().map(certificate => (
+            <div key={`key-cert-${certificate.key}`}>
+              <br />
+              <Radio
+                isChecked={tlsCertificate === certificate.value}
+                key={certificate.key}
+                onChange={onCertificateChange}
+                name={`radio-${certificate.key}`}
+                label={certificate.label}
+                id={`radio-${certificate.key}`}
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/modules/project/components/EndpointConfiguraion/EndpointConfiguraion.tsx:171]: Rename the ID.

File: console/console-init/ui/src/components/FinishedStep/FinishedStep.tsx [hunk 4]
```diff
@@ -48,26 +52,21 @@ const FinishedStep: React.FunctionComponent<IFinishedStepProps> = ({
...
+            icon={ExclamationCircleIcon}
             className={css(styles.cog_red_color)}
           />
           <Title headingLevel="h5" size="xl">
-            Creation error
+            The project cannot be created
           </Title>
           <br />
           <EmptyStateBody>
             Error occured while {projectType} Project creation for management,
```
Comment (anukritijha) [inline console/console-init/ui/src/components/FinishedStep/FinishedStep.tsx:63]: Replacing 'while' with 'during' makes more sense.

**Pull request merged**